packages/block-editor/src/components/copy-handler/README.md

```diff
@@ -0,0 +1,42 @@
+# Copy Handler
+
+The `CopyHandler` component handles the copy, cut and paste events of its children blocks. It handles multiple selection of blocks as well.
+
+Concretely, it handles the display of success messages and takes care of copying the block to the clipboard using the [raw handling API](https://github.com/WordPress/gutenberg/tree/master/packages/blocks/src/api/raw-handling).
+
+![Copy cut behaviours](https://make.wordpress.org/core/files/2020/09/copy-cut-behaviour.gif)
```

Comment:
Two notes:

* The GIF itself doesn't clearly show what is meant to happen. Perhaps [this one from the original pull request](https://user-images.githubusercontent.com/150562/81698101-6e341d80-945d-11ea-9bfb-b20781f55033.gif) would be better.

* _Copy cut behaviours_ should be _Copy/cut behaviours_ or _Copy and cut behaviours_.